vod/media_set_parser.c

```diff
@@ -2865,12 +2870,16 @@ media_set_parse_json(
 				parse_all_clips = FALSE;
 			}
 
+			// applying live window with infinite duration is necessary for playlistType event
+			// because it ensures that incomplete segments are not included in the resulting playlist
```

Comment:
I don't think it's about incomplete segments, it's about the fact we are not allowed to remove segments from the head of the media playlist when the type is event